google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/TypeProtos.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2;
+
+public class TypeProtos {
```

Comment:
`@InternalApi`? Is this only used in tests? `@VisibileForTesting`?

Reply:
does it matter since this is already in the test module?

Reply:
Oh, nvm, I thought it's under src/, ignore my comment!